frontends/web/src/locales/en/app.json

```diff
@@ -1655,6 +1655,20 @@
       "up-to-date": "Your app is up to date",
       "version": "App Version"
     },
+    "notes": {
+      "export": {
+        "description": "Create a backup of your transaction notes and account names.",
+        "success": "Notes successfully exported.",
+        "title": "Export notes"
+      },
+      "import": {
+        "description": "Restore your transaction notes and account names from a previously made backup file.",
+        "success": "Imported {{accountCount}} account names and {{transactionCount}} transaction notes",
```

Comment:
I'll take a look, such things would be nice to have right imo.

Reply:
Alternative text discussed with @thisconnect - don't show the number of notes/account names imported (is it needed?)

Just show generic message instead:

```
Successfully imported transaction notes and account names.
```

Reply:
thanks 👍 alternatively we can ofc always show a more generic message as you suggested.

Reply:
The issue is that the user could be importing a notes file that contains either completely different accounts than the wallet they are importing into, or some of the accounts in the import are not part of the target wallet. So the import is "successful" even if nothing was imported and there are un-imported notes in the file.

I put a msg like this so the user has a sense of what is going on. Maybe it can be reworded to make that point clear, or even extend it with something like:

"Notes of X accounts were not imported because they are not currently connected/remembered".

 @jadzeidan wdyt?

Reply:
this is the current version: 
<img width="615" alt="Screenshot 2024-06-10 at 12 13 32" src="https://github.com/benma/bitbox-wallet-app/assets/546900/cdf54a69-3b67-4062-9545-fb50382740f9">
<img width="586" alt="Screenshot 2024-06-10 at 12 12 49" src="https://github.com/benma/bitbox-wallet-app/assets/546900/f071c14f-1ccf-40c7-8fed-8cefebbe9d6a">
<img width="625" alt="Screenshot 2024-06-10 at 12 13 43" src="https://github.com/benma/bitbox-wallet-app/assets/546900/923a55cc-2fd9-4a86-8bb8-3fbbc36ed4ae">

Reply:
as discussed with @jadzeidan 
<img width="580" alt="Screenshot 2024-06-10 at 14 03 23" src="https://github.com/benma/bitbox-wallet-app/assets/546900/d088e19a-ae91-4bba-849e-7386450a6b17">


<img width="617" alt="Screenshot 2024-06-10 at 14 04 43" src="https://github.com/benma/bitbox-wallet-app/assets/546900/4f1f4d21-018e-4466-bb69-f92b7a0de36d">
<img width="626" alt="Screenshot 2024-06-10 at 14 05 40" src="https://github.com/benma/bitbox-wallet-app/assets/546900/ea3e71b7-9d1d-4f83-85a0-49c3a08d5478">

<img width="607" alt="Screenshot 2024-06-10 at 14 03 58" src="https://github.com/benma/bitbox-wallet-app/assets/546900/97d97eb8-2a03-42cc-82a1-07da1a170c6c">
<img width="639" alt="Screenshot 2024-06-10 at 14 04 18" src="https://github.com/benma/bitbox-wallet-app/assets/546900/56fa42db-0b9b-42da-8e03-c4f268e9222c">